content/src/outline.rs

```diff
@@ -496,6 +566,64 @@ impl Contour {
         const EPSILON: f32 = 0.001;
     }
 
+    /// Push a SVG arc
+    ///
+    /// Draws an ellipse section with radii given in `radius` rotated by `x_axis_rotation` to 'to' in the given `direction`.
+    /// If `large_arc` is true, draws an arc bigger than a 180°, otherwise smaller than 180°.
+    /// note that `x_axis_rotation` is in radians.
+    pub fn push_svg_arc(&mut self, radius: Vector2F, x_axis_rotation: f32, large_arc: bool, direction: ArcDirection, to: Vector2F) {
```

Comment:
How does this compare to the arc implementation that `pathfinder_canvas` uses?

Reply:
Path2D::ellipse_section takes
- center
- axis
- start_angle
- end_angle: f32,
- direction

svg_arc takes 
 - last point (implicit)
 - radius
 - axis rotation
 - large arc flag, because there are two possible circles
 - destination point